SpatialGDK/Source/SpatialGDK/Public/Interop/CrossServerRPCHandler.h

```diff
@@ -17,18 +17,21 @@ class CrossServerRPCHandler
 public:
 	CrossServerRPCHandler(ViewCoordinator& Coordinator, TUniquePtr<RPCExecutorInterface> RPCExecutor);
 
-	void ProcessOps(const TArray<Worker_Op>& WorkerMessages);
+	void ProcessMessages(const TArray<Worker_Op>& WorkerMessages, float DeltaTime);
 	void ProcessPendingCrossServerRPCs();
 	const TMap<Worker_EntityId_Key, TArray<FCrossServerRPCParams>>& GetQueuedCrossServerRPCs() const;
+	int32 GetRPCGuidsInFlightCount() const;
+	int32 GetRPCsToDeleteCount() const;
 
 private:
-	ViewCoordinator& Coordinator;
+	ViewCoordinator* Coordinator;
 	TUniquePtr<RPCExecutorInterface> RPCExecutor;
 	TSet<uint32> RPCGuidsInFlight;
-	TArray<TTuple<FDateTime, uint32>> RPCsToDelete;
+	TArray<TTuple<double, uint32>> RPCsToDelete;
 
-	double TimeElapsedS = 0.0;
-	double CommandRetryTime = 0.0;
+	const double GuidTimeout = 5.f;
```

Comment:
This should probably be static. Can also make this constant in a static namespace in the source file if easier.